### PR TITLE
use weaviate new python client and update readme

### PR DESCRIPTION
**Pull request by aaalexlit**
Contribution to #1 First phase.

What's new:
- use weaviate new python client
- use streamlit UI
- update README
- bump up some requirements versions

@iamleonie please review when you can

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,51 @@
...
 
 ## Setup instructions 
-1. Run the command 'pip install -r requirements.txt' in the directory to install all required dependencies 
-2. Make sure the docker file is running 
-3. Run the add_data.py to create schema and add data 
-4. Then run script.py and enter text in the input field. Then click classify to classify the comment as toxic or Non-toxic.
-
-## Usage instructions
-(TO DO)
+1. Clone the project
```
Comment (databyjp) [inline README.md]:
```suggestion
1. Clone the repository
```

File: README.md [hunk 2]
```diff
@@ -2,24 +2,51 @@
...
+    ```
+1. Run the Streamlit demo
+   ```shell
+   streamlit run app.py
+   ```
 
 ## Dataset license
 
-The dataset used for this example is available here: https://www.kaggle.com/datasets/akashsuper2000/toxic-comment-classification
+The dataset used for this example is available on Kaggle:
```
Comment (databyjp) [inline README.md:54]:
```suggestion
The dataset used for this example comes from Kaggle: 
```

File: add_data.py [hunk 3]
```diff
@@ -1,49 +1,60 @@
...
+    client.collection.delete(COLLECTION_NAME)
+
+# Create collection
+client.collection.create(
+    name=COLLECTION_NAME,
+    properties=[
+        wvc.Property(
+            name="comment",
+            data_type=wvc.DataType.TEXT,
+            descpirtion="The text of the comment"
```
Comment (databyjp) [inline add_data.py]: I notice that there is a typo here. Does this throw an error?

```suggestion
            description="The text of the comment"
```

File: add_data.py [hunk 4]
```diff
@@ -1,49 +1,60 @@
...
+        ),
+        wvc.Property(
+            name="label",
+            data_type=wvc.DataType.TEXT,
+            descpirtion="The label of the comment"
```
Comment (databyjp) [inline add_data.py]:
```suggestion
            description="The label of the comment"
```

File: add_data.py [hunk 5]
```diff
@@ -1,49 +1,60 @@
...
-    if(data.iloc[i][1]==1):
-        lab="toxic"
-    else:
-        lab="Non Toxic"
-    obj = {
-                "comment": str(data.iloc[i][0]),
-                "label": lab
-          }
-    client.data_object.create(obj, "Comments")
+if response.has_errors:
```
Comment (databyjp) [inline add_data.py:56]: Ah very nice to see this being used in the wild :D

**Review by databyjp — COMMENTED** [review]
Looks good to me (at least by eye) :). Great stuff!

I have left a couple of short comments. I am curious to see whether the typo in 'description' causes an error.

**Comment by iamleonie** [comment]
Hi @aaalexlit, just a quick update on the review status. I am running into the following error when I follow the instructions:

> weaviate.exceptions.WeaviateStartUpError: Weaviate did not start up in 5 seconds. Either the Weaviate URL http://localhost:8080 is wrong or Weaviate did not start up in the interval given in 'startup_period'.

I will have a closer look in the next few days and will be getting back to you. So, it's in progress. :)

File: add_data.py (hunk 3 above)
Comment (aaalexlit) [reply add_data.py]: thanks! 
no, it doesn't throw an error

**Review by iamleonie — APPROVED** [review]

**Comment by iamleonie** [comment]
Thank you for your contribution! It worked like a charm! (PS: Yes, I did have something already running on that port. Apologies for that.)

**Pull request merged**